packages/votingV2/src/mappings/votingV2.ts

```diff
@@ -311,6 +311,95 @@ function updateUsersSlashingTrackers(event: PriceResolved): void {
   globals.save();
 }
 
+function processSlashesInSameTransaction(
```

Comment:
This function fixes the problem where multiple slashes from multiple resolved prices requests in the same TX are combined.

This images illustrates the problem (left) and the fix after having this function (right):
<img width="800" alt="image" src="https://user-images.githubusercontent.com/53186719/197770870-1265a9d3-77a7-44a6-9964-8523a6aa5cd3.png">

<img width="800" alt="image" src="https://user-images.githubusercontent.com/53186719/197771013-64c6866a-fbd3-46e3-aaf2-3bafe3a0f582.png">